store/localstore/txn.go

```diff
@@ -190,11 +191,16 @@ func (txn *dbTxn) doCommit() error {
 	}
 
 	// Check dirty store
+	curVer, _ := globalVerProvider.GetCurrentVer()
 	err := txn.each(func(iter iterator.Iterator) error {
+		metaKey := codec.EncodeBytes(nil, iter.Key())
+		// put dummy meta key, write current version
+		b.Put(metaKey, codec.EncodeUint(nil, curVer.Ver))
+		mvccKey := MvccEncodeVersionKey(iter.Key(), curVer)
 		if len(iter.Value()) == 0 { // Deleted marker
-			b.Delete(iter.Key())
+			b.Put(mvccKey, tombstone)
```

Comment:
how about set tombstone in above Set, is it allowed to put tombstone as the valid value in Set?